docs/dev/adding_instruments.rst

```diff
@@ -187,92 +227,78 @@ In addition, any entries in ``**kwargs**`` take precedence, so if they need to,
+
+The :func:`Instrument.measurement <pymeasure.instruments.Instrument.measurement>` function returns a property that can only read values from an instrument.
+For example, if our "Extreme 5000" has the :code:`*IDN?` command we can write the following property to be added after the :code:`def __init__` line in our above example class, or added to the class after the fact as in the code here:
 
 .. _Python properties: https://docs.python.org/3/howto/descriptor.html#properties
 
 .. testcode::
 
-     Extreme5000.id = Instrument.measurement(
-        "*IDN?", """ Reads the instrument identification """
+     Extreme5000.cell_temp = Instrument.measurement(
+        ":TEMP?",
+        """Measure the temperature of the reaction cell.""",
      )
 
 .. testcode::
     :hide:
     
-    # We are not mocking this in FakeInstrument, let's override silently
-    Extreme5000.id = 'Extreme 5000 identification from instrument'
+    # We have to fake this silently because the FakeInstrument cannot do
+    # a measurement property, it only mirrors values that you sent first.
+    Extreme5000.cell_temp = 127.2
     
-You will notice that a documentation string is required, and should be descriptive and specific.
+You will notice that a documentation string is required, see :ref:`docstrings` for details.
 
-When we use this property we will get the identification information.
+When we use this property we will get the temperature of the reaction cell.
 
 .. doctest::
 
     >>> extreme = Extreme5000("GPIB::1")
-    >>> extreme.id           # Reads "*IDN?"
-    'Extreme 5000 identification from instrument'
+    >>> extreme.cell_temp  # Sends ":TEMP?" to the device
+    127.2
 
 The :func:`Instrument.control <pymeasure.instruments.Instrument.control>` function extends this behavior by creating a property that you can read and set. For example, if our "Extreme 5000" has the :code:`:VOLT?` and :code:`:VOLT <float>` commands that are in Volts, we can write the following property.
 
 .. testcode::
 
     Extreme5000.voltage = Instrument.control(
         ":VOLT?", ":VOLT %g",
-        """ A floating point property that controls the voltage
-        in Volts. This property can be set.
-        """
+        """Control the voltage in Volts (float)."""
     )
 
-You will notice that we use the `Python string format`_ :code:`%g` to pass through the floating point.
+You will notice that we use the `Python string format`_ :code:`%g` to format passed-through values as floating point.
 
 .. _Python string format: https://docs.python.org/3/library/string.html#format-specification-mini-language
 
-We can use this property to set the voltage to 100 mV, which will execute the command and then request the current voltage.
+We can use this property to set the voltage to 100 mV, which will send the appropriate command and then requests the current voltage.
```

Comment:
Sorry for correcting again this sentence to make it more clear, that you describe two different actions of this `control`:
```suggestion
We can use this property to set the voltage to 100 mV, which will send the appropriate command, and then to request the current voltage:
```

Reply:
No worries, it's clearer.